src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/StaffRepository.java

```diff
@@ -30,4 +30,11 @@
 
     @Query("select s.staffId from Staff s where s.officerCode = :officerCode")
     Optional<Long> findStaffIdByOfficerCode(String officerCode);
+
+    @Query("select staff " +
+        "from Staff staff, StaffTeam staffTeam " +
+        "where staffTeam.staffId = staff.staffId " +
+        "and staffTeam.teamId = :teamId " +
+        "and (staff.endDate is null or staff.endDate > sysdate) ")
+    List<Staff> findStaffByTeamId(Long teamId);
```

Comment:
Is sysdate and Oracle function? We've use CURRENT_DATE in a couple of other places so maybe stick with that if it's equivilent?

Reply:
sysdate seems to work but I'll change to CURRENT_DATE